oauth2client/contrib/gce.py

```diff
@@ -85,32 +121,110 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
     """
 
-    @util.positional(2)
-    def __init__(self, scope='', **kwargs):
+    @util.positional(3)
```

Comment:
1. @nathanielmanistaatgoogle Can confirm but I don't think we are supposed to change the number in `positional` on a non-breaking release.
1. If this is going to change, we should delete it.